### PR TITLE
Add remove 3 milestone

### PR DESCRIPTION
**Pull request by D4nielJ**
# Hello reviewer :smiley: 
## To add

* Add and remove functionalities
* Clear all completed button
* Change descriptions of the tasks

## Requirements

* Remove all hardcoded items from the tasks array.
* Create a new JavaScript file for the new functionality.
* Implement a function for adding a new task (add a new element to the array).
* Implement a function for editing task descriptions.
* Implement a function for deleting a task (remove an element from the array).
* Implement a function for the "Clear all completed" button (use filter() method).
* By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
* Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
* All changes to the To Do List should be saved in local storage.

[Live demo for you ](https://deploy-preview-3--jolly-wing-fd7a03.netlify.app/?utm_source=github&utm_campaign=bot_dp) :bouquet:

### CODE REVIEW

File: src/app/utils/todoList.js [hunk 1]
```diff
@@ -6,60 +6,94 @@ class TodoList {
...
+    storage.saveData(this.arr);
   };
 
   init = () => {
     if (!storage.loadData()) {
       this.addTask('Open minimalist', true);
-      this.addTask('Sort the list by dragging');
+      this.addTask('Add a new task');
       this.addTask('Check your tasks done');
+      this.addTask('You can delete me');
```
Comment (ZawadiSumba66) [inline src/app/utils/todoList.js]: - [x] All hardcoded items from the tasks array should be removed as instructed in the requirements.

File: src/app/components/deleteTask.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+class DeleteTask {
+  addListener = () => {
+    const rmButton = document.querySelectorAll('.todo-list__delete');
+    rmButton.forEach((button) => {
+      button.addEventListener('click', () => {
+        const index = parseInt(button.parentElement.getAttribute('data-index'), 10);
+        this.deleteTask(index);
+      });
+    });
+  }
```
Comment (ZawadiSumba66) [inline src/app/components/deleteTask.js:16]: - [x] The delete task functionality is not working properly. A task persists even when I press the delete button. Kindly look into this

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @D4nielJ  👋 , 

Good job so far!
#### HIGHLIGHTS 🚀 
- Descriptive pull request. ✔️ 
- Well written commit messages ✔️ 
- Great UX design ✔️ 

However, there are some issues that you still need to work on to go to the next step but you are almost there! 💪 💪 

## Required Changes ♻️
- [x] Your README currently lacks the getting started part which should contain instructions on cloning it and running the app locally. Please fix this 🙏 

_Check the comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @D4nielJ, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**